docs/examples.md

```diff
@@ -94,3 +94,20 @@ let app = new EmberApp(defaults, {
   }
 });
 ```
+
+#### De-duplicating SVG ids:
+
+```javascript
+let app = new EmberApp(defaults, {
+  svgJar: {
+    strategy: 'inline',
```

Comment:
not sure why, but it only works with `inline` strategy on my end

Reply:
That's interesting. I'll take a look when I have a chance this week.